src/index.js

```diff
@@ -43,11 +56,11 @@ function IdTokenVerifier(parameters) {
 
   if (supportedAlgs.indexOf(this.expectedAlg) === -1) {
     throw new error.ConfigurationError(
-      'Algorithm ' +
+      'Signature algorithm of ' +
```

Comment:
The error message was taken from the table on the internal doc, so if it's wrong it means either the doc or the sample is also wrong. Which version of this string do we want to take? @lbalmaceda

Reply:
I've changed it to be the same as the sample, mostly likely the doc is wrong.

Reply:
The main diff is that one mentions "ID Token" and the other just says "token". It purely depends on where you are placing this. e.g.:
- If you have a class to do "JWT signature verification" it wouldn't care what kind of token it is as long as it is a signed JWT. 
- If this instead is inside an "ID token validator" class or the error thrown is of type "ID Token X Exception" then you can guess what is wrong and don't need the name and surname of the token to figure that out. 

I always assumed the doc was the final version of messages because I no longer committed to the sample repo. But I'm OK with either. Both transmit the message.